operations/configuration_management/Create_an_Image_Customization_CFS_Session.md

```diff
@@ -42,7 +50,7 @@ true
 cray cfs sessions create --name example \
     --configuration-name configurations-example \
     --target-definition image --format json \
-    --target-group Compute IMS_IMAGE_ID
+    --target-group Compute $IMS_IMAGE_ID
```

Comment:
In the context of image customization, is `--target-group Compute` also incorrect here? I know this a generic guide on how to customize images but wondering what I might be able to do about that?

Reply:
This content has diverged in the main branch. Maybe it would make sense to cherry-pick b01d606ffc715b02a590de0d755ba9963ce77b6f here.

Reply:
Looking closer at that commit I believe we definitely should not cherry-pick it to 1.3. It was not intended for 1.3 and appears to document features that are not in 1.3.

Therefore I think we should probably do something to explain that these are example values.

Reply:
I think I've settled on something that works a little bit better. As an added bonus it will cherry-pick slightly more cleanly (though still not without conflict) to `main`.